exercises/concept/dnd-char/.meta/exemplar.jl

```diff
@@ -0,0 +1,33 @@
+"""
+    ability()
+
+Return a valid ability value.
+"""
+function ability()
+    # While the exercise doesn't explicitely require it, we simulate 4 dice rolls and discard the smallest roll.
+    # A minimal solution would be ability() = rand(3:18)
+
+    # Throw 4 dice
+    dice = [rand(1:6) for i in 1:4]
+
+    # Sort the dice so that [2:end] excludes the lowest roll
+    sort!(dice)
+    sum(dice[2:end])
+end
+
+"""
+    modifier(ability)
+
+Return the modifier based on an ability value.
+"""
+modifier(ability) = floor(Int, (ability - 10) / 2)
```

Comment:
Could use `fld` here.

`fld(ability - 10, 2)`

It's a bit faster, but maybe obscure to students.

Reply:
```suggestion
modifier(ability) = fld(ability - 10, 2)
```

> It's a bit faster, but maybe obscure to students.

I think this late into the track progression, we can expect people to look up functions they don't recognise in the REPL. `floor(T, val)` isn't explained either.